tests/baselines/reference/systemModuleEnum.js

```diff
@@ -0,0 +1,25 @@
+//// [systemModuleEnum.ts]
+
+export enum Foo {}
+
+export module Bar {
+	export var baz = (): any => {}
+}
+
+
+//// [systemModuleEnum.js]
+System.register([], function(exports_1) {
+    var Foo, Bar;
+    return {
+        setters:[],
+        execute: function() {
+            (function (Foo) {
+            })(Foo || (Foo = {}));
+            exports_1("Foo", Foo);

```

Comment:
The compiler does not generate a semicolon at the end of this line, however it appears to be necessary.  I am *guessing* that the JS runtime is seeing an attempt to call a function on `exports_1("Foo", Foo)(...)`, since the following function is wrapped in parenthesis.  The semicolon tells the runtime that a new expression is starting on line 19.